src/forest.py

```
@@ -73,6 +74,8 @@ def clip_dem_with_polygon(
     """
     Создает буфер вокруг полигона и обрезает DEM по этому буферу.
     """
+    if not progress.update(10, "Создание буфера..."):
```

Ruff (0.11.9)

Undefined name `progress`

(F821)